src/sysupdate/sysupdate.c

```diff
@@ -666,20 +677,38 @@ static int context_show_version(Context *c, const char *version) {
         if (!have_sha256)
                 (void) table_hide_column_from_display(t, 12);
 
+        STRV_FOREACH(changelog, c->changelog) {
+                *changelog = strreplace(*changelog, "@v", version);
+                if (*changelog == NULL)
```

Comment:
according to our coding style do not use explicit comparisons with NULL to check for null pointers. instead rely on C's downgrade-to-bool feature for this

also, this leaks the old memory in case of oom.